### ISSUE TITLE
Problems about Focal loss addition 

### ISSUE THREAD

**Issue by songl17** [wannabeOG/Mask-RCNN#13]
Q1.In compute_loss( ) function, rpn_class_loss = utils.compute_rpn_class_loss(rpn_match, rpn_class_logits), but utils.compute_rpn_class_loss() doesn't exist.
Q2.In model.compute_rpn_class_loss(), you use the original cross-entropy instead. The only use of focal loss is in mask.forward(), but it seemed doesn't work.
Looking for your kind reply. Thanks in advance.
